.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,17 @@
+blank_issues_enabled: false
+contact_links:
+  - name: BigBlueButton Setup
```

Comment:
Here consider linking only to

```
- name: BigBlueButton GitHub Repository
  url: https://github.com/bigbluebutton/bigbluebutton
  about: Everything BigBlueButton (in case you are looking for something different than plugin sdk)
- name: Looking for commercial support in building/maintaining plugins
   url: https://bigbluebutton.org/commercial-support
   about: List of companies offering commercial BigBlueButton support
```

as the rest is generic BBB. Maybe also add

```
  - name: BigBlueButton Developers Mailing List:
    url: https://groups.google.com/forum/#!forum/bigbluebutton-dev:
    about: Ask questions and share experiences with other BigBlueButton developers (including about plugins).
```

(notice the plugins twist)